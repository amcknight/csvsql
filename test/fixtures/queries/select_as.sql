SELECT first_name AS name, 3 as three
FROM test/fixtures/tables/people
