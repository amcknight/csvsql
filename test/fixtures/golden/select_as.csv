name, three
Andrew, 3
Tom, 3
Cayley, 3
July, 3
